Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -104,17 +104,17 @@ - (void)setPlaceholderColor:(UIColor *)placeholderColor
 }
 
 #if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
-- (void)setSelectionColor:(UIColor *)selectionColor
+- (void)setSelectionColor:(RCTUIColor *)selectionColor
```

Comment:
Missing the TODO comment

Reply:
This change is already inside a ` // [TODO(macOS ISS#2323203)` ... ` // ]TODO(macOS ISS#2323203)` block.